src/services/InvoiceService.js

```diff
@@ -24,7 +24,16 @@ const createInvoice = async ( newInvoice ) => {
 */
 const getAllInvoices = async () => Invoice.find().exec();
 
+/**
+ * Gets a single invoice from the database
+ *
+ * @param {*} id the id of the invoice that will be retrieved
+ * @returns the invoice with the given id
+ */
+const getInvoiceById = async id => Invoice.findById( id ).exec();
```

Comment:
Use .populate() to automatically populate invoice items array

Syntax should be something like this:
```
Invoice.findById( id ).populate( {
    path: "invoiceItems",
} ).exec();
```